google-cloud-bigtable/src/test/java/com/google/cloud/bigtable/admin/v2/it/BigtableBackupIT.java

```diff
@@ -0,0 +1,358 @@
+import com.google.cloud.bigtable.admin.v2.models.CreateBackupRequest;
+import com.google.cloud.bigtable.admin.v2.models.CreateTableRequest;
+import com.google.cloud.bigtable.admin.v2.models.RestoreTableRequest;
+import com.google.cloud.bigtable.admin.v2.models.RestoredTableResult;
+import com.google.cloud.bigtable.admin.v2.models.Table;
+import com.google.cloud.bigtable.admin.v2.models.UpdateBackupRequest;
+import com.google.cloud.bigtable.data.v2.BigtableDataClient;
+import com.google.cloud.bigtable.data.v2.BigtableDataSettings;
+import com.google.cloud.bigtable.data.v2.models.RowMutation;
+import com.google.common.base.Joiner;
+import com.google.common.base.MoreObjects;
+import com.google.common.collect.Lists;
+import com.google.protobuf.Timestamp;
+import io.grpc.StatusRuntimeException;
+import java.io.IOException;
+import java.util.List;
+import java.util.Random;
+import java.util.concurrent.ExecutionException;
+import java.util.concurrent.TimeUnit;
+import java.util.concurrent.TimeoutException;
+import java.util.logging.Logger;
+import org.junit.*;
+import org.threeten.bp.Duration;
+import org.threeten.bp.Instant;
+
+public class BigtableBackupIT {
+  private static final Logger LOGGER = Logger.getLogger(BigtableBackupIT.class.getName());
+
+  private static final String PROJECT_PROPERTY_NAME = "bigtable.project";
+  private static final String INSTANCE_PROPERTY_NAME = "bigtable.instance";
+  private static final String CLUSTER_PROPERTY_NAME = "bigtable.cluster";
+  private static final String ADMIN_ENDPOINT_PROPERTY_NAME = "bigtable.adminendpoint";
+  private static final String DATA_ENDPOINT_PROPERTY_NAME = "bigtable.dataendpoint";
+  private static final String TABLE_SIZE_PROPERTY_NAME = "bigtable.tablesizekb";
+  private static final int[] BACKOFF_DURATION = {2, 4, 8, 16, 32, 64, 128, 256, 512, 1024};
+
+  private static final String TEST_TABLE_SUFFIX = "test-table-for-backup-it";
+  private static final String TEST_BACKUP_SUFFIX = "test-backup-for-backup-it";
+
+  private static final int DAYS_IN_SECONDS = 24 * 60 * 60;
+
+  private static BigtableTableAdminClient tableAdmin;
+  private static BigtableDataClient dataClient;
+
+  private static String targetProject;
+  private static String targetInstance;
+  private static String targetCluster;
+  private static Table testTable;
+  private static String prefix;
+
+  @BeforeClass
+  public static void createClient()
+      throws IOException, InterruptedException, ExecutionException, TimeoutException {
+    List<String> missingProperties = Lists.newArrayList();
+
+    targetProject = System.getProperty(PROJECT_PROPERTY_NAME);
+    if (targetProject == null) {
+      missingProperties.add(PROJECT_PROPERTY_NAME);
+    }
+
+    targetInstance = System.getProperty(INSTANCE_PROPERTY_NAME);
+    if (targetInstance == null) {
+      missingProperties.add(INSTANCE_PROPERTY_NAME);
+    }
+
+    targetCluster = System.getProperty(CLUSTER_PROPERTY_NAME);
```

Comment:
Does this need to be specified? Can it just be the first cluster in the instance?

Reply:
I discussed this with @kolea2 and integration test improvements will be in separate PRs.